### PR TITLE
Add Community Meetings Component

### PR DESCRIPTION
**Pull request by cgiradkar**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@ yarn-error.log
 .docusaurus/
 build/
 types
+package-lock.json
+yarn.lock
```
Comment (Luap99) [inline .gitignore:19]: Why did you add these to gitignore?
AFAIK it is important to keep these in the repo to make sure all are using the same dependency versions.

see https://yarnpkg.com/getting-started/qa#should-lockfiles-be-committed-to-the-repository

**Comment by TomSweeneyRedHat** [comment]
I can barely spell tsx.  I saw the files for the notes were split up into individual directories with an index.md, that works for me.  Is there a location in the .tsx that needs to be updated when a new one is added, or does that happen auto-magically?  Also, is there a list that some one can get to to see all of the old notes?

File: .gitignore (hunk 1 above)
Comment (cgiradkar) [reply .gitignore:19]: Thanks for the article, last I read about those issues were ages ago and now the lock files are needed to be added. I will do so and push the changes.

**Comment by cgiradkar** [comment]
> I can barely spell tsx. I saw the files for the notes were split up into individual directories with an index.md, that works for me. Is there a location in the .tsx that needs to be updated when a new one is added, or does that happen auto-magically? Also, is there a list that some one can get to to see all of the old notes?

The tsx file is just there for creating a central file so that the imports will be done at a single point.
While adding new files, please follow following steps to add the md files smoothly:

1. Add the folder with similar nomenclature (containing index.md file)
2. Open static/data/meetings/notes/index.ts and add your import anywhere in the file.
3. In the export section, add the name of the import.

PS: the nomenclature used for exporting is as follows:
the directory name: 2021-12-16 => export name used: "F<folder name without the dash>".
You can use any nomenclature, just use the same to export/import as well.

**Comment by Luap99** [comment]
Just a general question, is it correct that the real md file is not served anywhere. Seems like your component links to the hackmd link, correct? If we already import the raw md file I think it would make sense to render them directly on our site without linking them to another side.

For the process of adding new stuff I will defer that judgement to @TomSweeneyRedHat . I think these three listed steps are easy enough.

---
On another note in your git sign off you use the github name but it should use your real name instead.
ref: https://github.com/containers/podman/blob/main/CONTRIBUTING.md#sign-your-prs

File: src/components/layout/CommunityMeetingsCardGrid/index.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+              title={card?.title}
+              subtitle={card?.date}
+              details={card?.timeZone}
+              text={card?.subtitle}
+              data={card?.buttons}
+              primary={true}
+            />
+            <SectionHeader
+              title=""
+              description="Most Recent Past meetings"
```
Comment (Luap99) [inline src/components/layout/CommunityMeetingsCardGrid/index.tsx:172]: This should say just `Past meetings` IMO.

**Comment by cgiradkar** [comment]
> Just a general question, is it correct that the real md file is not served anywhere. Seems like your component links to the hackmd link, correct? If we already import the raw md file I think it would make sense to render them directly on our site without linking them to another side.
> 
> For the process of adding new stuff I will defer that judgement to @TomSweeneyRedHat . I think these three listed steps are easy enough.

I can make a modal which will show the md file in react format.

File: src/components/layout/CommunityMeetingsCardGrid/index.tsx (hunk 2 above)
Comment (cgiradkar) [reply src/components/layout/CommunityMeetingsCardGrid/index.tsx:172]: I can change that,but the UX has `Most Recent Past meetings`. Which one should I put?

**Review by Luap99 — COMMENTED** [review]
Also can you invert he order of the dropdown? The most recent meeting should always be at top not bottom IMO ,as this makes more sense if the lists gets larger.

**Comment by cgiradkar** [comment]
> Also can you invert he order of the dropdown? The most recent meeting should always be at top not bottom IMO ,as this makes more sense if the lists gets larger.

Will do

File: src/components/layout/CommunityMeetingsCardGrid/index.tsx (hunk 2 above)
Comment (Luap99) [reply src/components/layout/CommunityMeetingsCardGrid/index.tsx:172]: @mairin @TomSweeneyRedHat WDYT?
Maybe it is my language barrier but for me this can either be `most recent` or `past` but not both at the same time.

Comment (cgiradkar) [reply src/components/layout/CommunityMeetingsCardGrid/index.tsx:172]: agreed. was just confirming as here the @mairin gave me a UX to work with: 
Screenshot from 2023-06-16 14-55-19

Comment (TomSweeneyRedHat) [reply src/components/layout/CommunityMeetingsCardGrid/index.tsx:172]: "Most Recent" please.  That should have the past 2 to 4 meetings with links to the notes/videos.  There should be a spot somewhere to see "Older" meetings/videos where you'd see all the meetings in order, the most recent at the top of the list.

Comment (TomSweeneyRedHat) [reply src/components/layout/CommunityMeetingsCardGrid/index.tsx:172]: And I should add that I'm absolutely fine if the page with all of the meetings is very plain and vanilla.  In reality, I'll probably be one of the few people going to it.

**Comment by TomSweeneyRedHat** [comment]
@cgiradkar I've added the May and June notes to the old site via: https://github.com/containers/podman.io_old/pull/610.  The new files there are:

```
	new file:   community/meeting/notes/2023-05-18/index.md
	new file:   community/meeting/notes/2023-06-06/index.md
	new file:   community/meeting/notes/2023-06-15/index.md
```

I did not update the menu page that listed all of the meeting notes.  We are all caught up to date at this point.

**Comment by rhatdan** [comment]
@Luap99 @TomSweeneyRedHat is this ready to merge?

**Comment by cgiradkar** [comment]
> @Luap99 @TomSweeneyRedHat is this ready to merge?

@rhatdan I have pushed latest changes as per expectations. once @Luap99 and @mairin gives a go-ahead, we can merge this PR.
PS: @TomSweeneyRedHat I will merge the other PR for inclusion of latest Notes soon after this PR is merged

File: package-lock.json
Comment (Luap99) [inline package-lock.json:1]: where does this file come from?
I assume we just use `yarn` in this repo so I wouldn't expect an npm file here? At least if I only use yarn this file is not generated.

**Review by Luap99 — COMMENTED** [review]
Thanks this looks good mostly but there is one problem:
image
The Meeting Minutes button there cannot be clicked, nothing happens in this case.

File: package-lock.json
Comment (cgiradkar) [reply package-lock.json:1]: i added those files locally in a static folder, which Docusaurus  copies as is

Comment (cgiradkar) [reply package-lock.json:1]: I used npm initially as yarn was having some issues on my machine. Removed the package-lock.json after use. added back due to PR comment but will remove again as discussed here.

**Comment by TomSweeneyRedHat** [comment]
FWIW, LGTM once the issue @Luap99 found is hunted down.

**Review by Luap99 — COMMENTED** [review]
LGTM
